style.css

```diff
@@ -0,0 +1,303 @@
+    width: 137px;
+    height: 52px;
+    background: #1BA0AB;
+    border-radius: 5px;
+    color: white;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 22px;
+    letter-spacing: 0.2px;
+    text-align: center;
+}
+
+/* Header */
+
+header {
+    height: 100vh;
+    background: #FFF5F0;
+    background-position: bottom;
+    background-size: cover;
+    display: flex;
+    align-items: flex-end;
+}
+
+
+.header-content {
+    margin-bottom: 30px;
+    margin-left: 9rem;
+}
+
+.name-message {
+    font-weight: 700;
+    font-style: normal;
+    font-size: 16px;
+    line-height: 24px;
+    letter-spacing: 0.1px;
+    color: #1BA0AB;
+}
+
+
+.header__head {
+    font-weight: 700;
+    font-size: 58px;
+    line-height: 80px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+    margin: 30px 0;
+}
+
+
+.header__desc {
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 30px;
+    letter-spacing: 0.2px;
+    color: #737373;
+    margin: 30px 0;
+}
+
+.link-header__btn {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    gap: 10px;
+    align-items: center;
+}
+
+.btn--outlined {
+    border: 1px solid #1BA0AB;
+    border-radius: 5px;
+    color: #1BA0AB;
+    background-color: transparent;   
+}
+
+
+
+/* Main */
+
+.main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-top: 5rem;
+    gap: 5rem;
+}
+
+.main__content {
+    display: flex;
+    flex-direction: column;
+    gap: 30px;
+}
+
+.main__line {
+    width: 94px;
+    height: 7px;
+    background-color: #E74040;
+}
+
+.title {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 50px;
+    letter-spacing: 0.2px;
+    color: #252B42;
+}
+
+.text {
+    font-style: 400;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 20px;
+    letter-spacing: 0.2px;
+    color: #737373;
+}
+
+.link--highlighted {
+    text-decoration: none;
+    color: #1BA0AB;
+}
+
+
+/* Container */
+
+.container__content {
+    padding-left: 10rem;
+    padding-bottom: 5rem;
+    padding-top: 10rem;
+}
+
+.descr {
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 24px;
+    letter-spacing: 0.2px;
+    color: #1BA0AB;
+}
+
+.title__style{
+    padding: 13px 0;
+}
+
+.card {
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+}
+
+.container__card {
+    background: #FFFFFF;
+    box-shadow: 0px 13px 19px rgba(0, 0, 0, 0.07);
+    border-radius: 20px;
+    overflow: hidden;
+}
+
+.name-message--style {
+    text-align: center;
+    color: #252B42;
+    padding-top: 20px;
+}
+
+.card__prof {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 16px;
+    letter-spacing: 0.2px;
+    text-align: center;
+    color: #737373;
+    padding: 10px 0;
+
+}
+
+.card__footer {
+    text-align: center;
+    margin-bottom: 2rem;
+}
+
+.fa-brands {
+    color: #1BA0AB;
+    font-size: 22px;
+    padding: 0 10px;
+}
+
+
+/* Footer */
+
+footer {
```

Comment:
## style wrong
`footer` this is not bem

Reply:
What he means is that you are not allowed to style with custom tags when using BEM.